UI/window-basic-settings-stream.cpp

```diff
@@ -576,11 +596,18 @@ void OBSBasicSettings::on_authPwShow_clicked()
 OBSService OBSBasicSettings::SpawnTempService()
 {
 	bool custom = IsCustomService();
-	const char *service_id = custom ? "rtmp_custom" : "rtmp_common";
+	bool webrtc = IsWebRTC();
+	const char *service_id = "rtmp_common";
+
+	if (custom) {
+		service_id = "rtmp_custom";
+	} else if (webrtc) {
+		service_id = "webrtc";
```

Comment:
We should probably follow the behavior of RTMP and name this something like `whip_custom`.

Reply:
@tt2468 I am not sure what the best thing to do with the name `WHIP` vs `WebRTC` is.

I believe `webrtc_custom` is a more intuitive name. WHIP is one piece of technology we are using, but WebRTC is the entire thing.

It will also be easier for users to understand. WebRTC is a common term, WHIP is something only WebRTC implementors are really concerned with.

Reply:
So with that do you mind if I change this to `webrtc_custom` instead and keep the name `webrtc` for the plugin?